<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>308</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1370</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic/resource.qrc">:/pic/Pic.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="Add_action"/>
    <addaction name="dispose_action"/>
    <addaction name="action_addcsv"/>
    <addaction name="action_TrainnigModel"/>
    <addaction name="action_FaceRecognition"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="actionData"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_use"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>796</width>
     <height>34</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>796</width>
     <height>34</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Add_action"/>
   <addaction name="dispose_action"/>
   <addaction name="action_addcsv"/>
   <addaction name="action_TrainnigModel"/>
   <addaction name="action_FaceRecognition"/>
   <addaction name="action_about"/>
   <addaction name="action_use"/>
   <addaction name="actionData"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Add_action">
   <property name="icon">
    <iconset resource="pic/resource.qrc">
     <normaloff>:/pic/add.png</normaloff>:/pic/add.png</iconset>
   </property>
   <property name="text">
    <string>添加照片</string>
   </property>
  </action>
  <action name="dispose_action">
   <property name="icon">
    <iconset resource="pic/resource.qrc">
     <normaloff>:/pic/PrintSc.png</normaloff>:/pic/PrintSc.png</iconset>
   </property>
   <property name="text">
    <string>照片处理</string>
   </property>
  </action>
  <action name="action_addcsv">
   <property name="icon">
    <iconset resource="pic/resource.qrc">
     <normaloff>:/pic/addCSV.png</normaloff>:/pic/addCSV.png</iconset>
   </property>
   <property name="text">
    <string>增添csv</string>
   </property>
  </action>
  <action name="action_TrainnigModel">
   <property name="icon">
    <iconset resource="pic/resource.qrc">
     <normaloff>:/pic/train.png</normaloff>:/pic/train.png</iconset>
   </property>
   <property name="text">
    <string>模型训练</string>
   </property>
  </action>
  <action name="action_FaceRecognition">
   <property name="icon">
    <iconset resource="pic/resource.qrc">
     <normaloff>:/pic/recognition.png</normaloff>:/pic/recognition.png</iconset>
   </property>
   <property name="text">
    <string>人脸识别</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="pic/resource.qrc">
     <normaloff>:/pic/about.png</normaloff>:/pic/about.png</iconset>
   </property>
   <property name="text">
    <string>关于程序</string>
   </property>
  </action>
  <action name="action_use">
   <property name="icon">
    <iconset resource="pic/resource.qrc">
     <normaloff>:/pic/use.png</normaloff>:/pic/use.png</iconset>
   </property>
   <property name="text">
    <string>程序使用</string>
   </property>
  </action>
  <action name="actionData">
   <property name="icon">
    <iconset resource="pic/resource.qrc">
     <normaloff>:/pic/check.png</normaloff>:/pic/check.png</iconset>
   </property>
   <property name="text">
    <string>Data文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic/resource.qrc"/>
 </resources>
 <connections/>
</ui>
